feeds/src/state/ducks/aggregator/selectors.ts

```diff
@@ -26,10 +64,10 @@ const oracles = createSelector(
     if (!list) return []
 
     const result = list
-      .map(a => {
+      .map((address: OracleNode['address']) => {
         return {
-          address: a,
-          name: upcaseOracles[a.toUpperCase()] || 'Unknown',
+          address,
+          name: upcaseOracles[address] || 'Unknown',
```

Comment:
I know this wasn't a change you made, but the name of this variable is a little confusing given there is a function with the name name. How do you feel about calling the variable `upcasedOracles`?